### PR TITLE
Feature zeus/XOT-105 report opportunity button

### PR DESCRIPTION
**Pull request by stevenburnell-dit**
screen shot 2018-05-08 at 17 01 42

### CODE REVIEW

File: app/views/poc/components/_feedback.html.haml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.feedback
+  = link_to "Is there anything wrong with this page?", "https://contact-us.export.great.gov.uk/directory/"
```
Comment (agiamas) [inline app/views/poc/components/_feedback.html.haml]: can you please change this to Figaro.env.CONTACT_US_FORM ?

like this:
        =link_to 'let us know', "#{Figaro.env.CONTACT_US_FORM}"

File: app/views/poc/opportunities/show.html.haml [hunk 2]
```diff
@@ -81,6 +81,10 @@
                 = opportunity.country_guide_links
                 = opportunity.link_to_aid_funded "Your guide for aid funded business"
 
+            - if opportunity.source.eql? 'volume_opps'
+              -# TODO: Just sending to feedback location for now
+              = link_to "Report opportunity", "https://contact-us.export.great.gov.uk/directory/", :class=>"report"
```
Comment (agiamas) [inline app/views/poc/opportunities/show.html.haml]: same here

File: app/views/poc/components/_feedback.html.haml (hunk 1 above)
Comment (stevenburnell-dit) [reply app/views/poc/components/_feedback.html.haml]: Changed.

File: app/views/poc/opportunities/show.html.haml (hunk 2 above)
Comment (stevenburnell-dit) [reply app/views/poc/opportunities/show.html.haml]: Changed.

**Pull request merged**